### DISCUSSION TITLE
Request to install  mlmym on my lemmy instance

### DISCUSSION THREAD

**Discussion by kukumaka** [ubergeek77/Lemmy-Easy-Deploy#89] [General]
I am.

**Comment by ubergeek77** [comment]
Hello, I haven't seen this before but I just got a notification because you closed it. Sorry for missing it!

You can do this by manually editing the deployment via advanced configuration:

https://github.com/ubergeek77/Lemmy-Easy-Deploy/blob/main/ADVANCED_CONFIGURATION.md

I have no specific steps to share, but any Docker based image should work. You will need to remember to update your custom deployment every time the official deployment changes, although this happens very rarely.
